robber/helper.py

```diff
@@ -26,3 +26,41 @@ def build_called_params_string(call_args):
         match = re.search('call\((.*)\)', str(call_args))
         params = match.group(1)
         return params or 'no arguments'
+
+    @classmethod
+    def unicode_to_str(cls, obj):
```

Comment:
I just want to gather all the helper functions into one `Helper` class like how Python stores its settings. Is there any Pythonic thing that prevents me from doing so or it justs one's preference?